packages/compass-crud/src/components/crud-toolbar.tsx

```diff
@@ -194,7 +194,8 @@ const CrudToolbar: React.FunctionComponent<CrudToolbarProps> = ({
         </div>
         <div className={toolbarRightActionStyles}>
           <Body data-testid="crud-document-count-display">
-            {start} - {end} of {displayedDocumentCount}
+            {start} – {end}{' '}
+            {displayedDocumentCount && `of ${displayedDocumentCount}`}
```

Comment:
drive-by to hide `of` when refreshing Documents Tab.

Reply:
and uses `–` instead of `-`. we are using this on aggregations screen and it looks better. and that's why some changes in e2e tests